src/main/java/cz/xtf/radanalytics/oshinko/deployment/Oshinko.java

```diff
@@ -116,13 +119,13 @@ private static OshinkoPoddedWebUI deployWebUIPodCommonLogic(String templateName,
 	 *
 	 * @return path to Oshinko Webui resources yaml file
 	 */
-	private static String getOshinkoWebuiResources(String oshinkoWebUITemplate) {
-		log.info("Getting Oshinko Web UI Resources from temp directory");
+	public static String getOshinkoResources(String workDir, String oshinkoTemplate, String oshinko_resources_url) {
```

Comment:
please make this more generic - ie. remove "Oshinko" references and move to the TestHelper class, so it can be consumed in multiple ways